core/translate/mod.rs

```diff
@@ -595,6 +596,10 @@ fn update_pragma(
             query_pragma("journal_mode", header, program)?;
             Ok(())
         }
+        PragmaName::WalCheckpoint => {
+            // TODO
+            Ok(())
```

Comment:
todo: currently checkpoint pragma 

1. with body goes to `update_pragma`, 
2. without body goes to `query_pragma`